tests/layer_tests/tensorflow_tests/test_tf_IsInf.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+from common.utils.tf_utils import mix_array_with_value
+
+
+class TestIsInf(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'x' in inputs_info, "Test error: inputs_info must contain `data`"
+        x_shape = inputs_info['x']
+        inputs_data = {}
+        data = np.random.randint(-50, 50, x_shape).astype(np.float32)
+        inputs_data['x'] = mix_array_with_value(data, np.inf)
+        return inputs_data
+
+    def create_is_inf_net(self, x_shape, x_type):
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(x_type, x_shape, 'x')
+            tf.raw_ops.IsInf(x=x, name='is_inf')
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        return tf_net, None
+
+    test_data_basic = [
+        dict(x_shape=[8], x_type=tf.float32),
+        dict(x_shape=[3, 7], x_type=tf.float32),
+        dict(x_shape=[4, 3, 2], x_type=tf.float32),
+    ]
+
+    @pytest.mark.parametrize("params", test_data_basic)
```

Comment:
do we plan to enable it in precommit?

Reply:
Yes, once it is implemented in CPU

Reply:
What is the current test result? Do we use a reference to calculate values?

Reply:
No, we don't. They are failing with this error:

```
>       net = core.read_model(self.model, self.weights)
E       RuntimeError: Check 'false' failed at openvino\src\frontends\common\src\frontend.cpp:54:
E       Converting input model
E       Cannot create IsNaN layer is_nan id:1 from unsupported opset: opset10
```

Reply:
Should we temporary disable these tests with xfail and create a new ticket to enable it once it is implemented in CPU?

Reply:
Added `xfail` and refer to the CPU implementation. I will put a comment there to remove this `xfail` when they post PR.